File: app/src/main/java/com/example/quiz/PlayAgainActivity.java
package com.example.quiz;


import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class PlayAgainActivity extends Activity {

    Button playAgain;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_play_again);
        //Initialize
        playAgain = (Button) findViewById(R.id.playAgainButton);

        //play again button onclick listener
        playAgain.setOnClickListener(view -> {
            finish();
        });
    }

    @Override
    public void onBackPressed() {
        finish();
    }
}

File: app/src/main/java/com/example/quiz/PlayAgainActivity.kt
package com.example.quiz

import android.app.Activity
import android.os.Bundle
import android.view.View
import com.example.quiz.databinding.ActivityPlayAgainBinding

class PlayAgainActivity : Activity() {
    private lateinit var binding: ActivityPlayAgainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityPlayAgainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        binding.playAgainButton.setOnClickListener { view: View? -> finish() }
    }

    override fun onBackPressed() {
        finish()
    }
}